packages/core/src/Configs/MintV3PoolConfig.class.ts

```diff
@@ -0,0 +1,87 @@
+import { AssetAmount, IAssetAmountMetadata } from "@sundaeswap/asset";
+import { IMintV3PoolConfigArgs } from "../@types/index.js";
+
+import { TFee } from "../@types/queryprovider.js";
+import { Config } from "../Abstracts/Config.abstract.class.js";
+
+export class MintV3PoolConfig extends Config<IMintV3PoolConfigArgs> {
+  assetA?: AssetAmount<IAssetAmountMetadata>;
+  assetB?: AssetAmount<IAssetAmountMetadata>;
+  fees?: TFee;
+  marketTimings?: [bigint, bigint];
```

Comment:
shouldn't we expect `marketTimings` to be of type `[number, number]` since we'd pass milliseconds?

Also, in your example in the PR description, you're passing numbers instead of bigints:

```ts
const fiveDaysLater = Date.now() + 1000 * 60 * 60 * 24 * 5;
await SDK.builder(EContractVersion.V3)
  .mintPool({
     assetA: new AssetAmount(200_000_000_000n, ADA_METADATA),
     assetB: new AssetAmount(200_000_000_000n, OTHER_METADATA),
     fees: [5n, 10n],
     marketTimings: [Date.now(), fiveDaysLater],
     ownerAddress: "addr123..." ,
     referralFee:  {
       destination: "addr1234...." ,
       payment: new AssetAmount(1_000_000n, {
         assetId: "",
         decimals: 6,
       }),
     },
  }
})
```

Reply:
It can be either/or, but will eventually be converted to `bigint` for the datum.